lib/tasks/safe_reset.rake

```diff
@@ -3,7 +3,7 @@
 namespace :db do
   desc "Remove all database content, then seed it. Only for dev environments."
   task safe_reset: :environment do
-    if Rails.env.development? || Rails.env.deployed_development?
+    if Rails.env.development? || Rails.env.deployed_development? || Rails.env.sandbox?
```

Comment:
Have you thought twice? 😛

Reply:
Could you elaborate on your concerns? I'm not sure what you're getting at

Reply:
I think it's a joke about the thing that the script prints out :D 

> puts "You should think twice before recreating staging or production database!"
puts "Fire off Rails console and copy the code from this task if you really have to."
puts "Make sure you know what you are doing and maybe do a back-up first."

Reply:
I haven't had a direct response so I'm going to assume Michal is right and  it's fine and go ahead and merge. We can always take it out again.